css/master.css

```diff
@@ -196,7 +257,7 @@
 
   100% {
     transform: rotateZ(360deg);
-    transition-duration: 1;
+    transition-duration: 1s;
   }
 }
 
```

Comment:
- [ ] Great job so far! 👏🏻  however according to the project requirement, on the desktop size, the hamburger menu button, the closed button, and the menu page are not visible. Kindly correct this. You can use your @media (min-width: 768px) to display none of your navigation menu.
![6666](https://user-images.githubusercontent.com/79058364/167599792-0f7b1273-f661-409c-9606-a2e20fb8c41c.png)